### DISCUSSION TITLE
Is this guide not worked on anymore ?

### DISCUSSION THREAD

**Discussion by AzhamProdLive** [suppressio/vr-pico4-steam-linux#3] [General]
Hello there ! New Pico4 owner there and I looked forward to find a great guide in order to play SteamVR games on Linux ! 
After stumbling on your guide, I was happy to see that this existed, but got quickly taken back by the fact it's not finished at all ...

My main question is : Are you still working a bit on this guide ? Or did you gave up on it ? I hope not the latter, but it's fine otherwise ! 

Have a great day :)

**Comment by suppressio** [comment]
Hi,

sorry for the late reply! You're right, the guide is incomplete and pretty outdated. But good news—I'm on vacation right now, so I'll have some time to work on it and get it updated for you.

Thanks for your patience

**Reply by AzhamProdLive** [reply]
> Hi,
> 
> sorry for the late reply! You're right, the guide is incomplete and pretty outdated. But good news—I'm on vacation right now, so I'll have some time to work on it and get it updated for you.
> 
> Thanks for your patience

Hey, 

All good for the late reply ! Better one day than never ! 
It's fine, enjoy your vacation and thanks a lot in advance :)

Have a nice holiday :D

**Comment by suppressio** [comment]
The guide has been updated to use ALVR instead of ALXR. This change simplifies the setup process and provides full audio support on Linux. The guide now relies solely on ALVR version 20.9.1, which resolves previous audio streaming issues. The old, incomplete guide using ALXR, which required additional software for audio streaming, is still available under the tag "old_alxr".
